cpp/doxygen/developer_guide/DEVELOPER_GUIDE.md

```diff
@@ -245,6 +246,27 @@ RMM provides a "default" memory resource for each device that can be accessed an
 respectively. All memory resource parameters should be defaulted to use the return value of
 `rmm::mr::get_current_device_resource()`.
 
+### Resource Refs
+
+Memory resources are passed via resource ref parameters. A resource ref is memory resource wrapper
+that enables consumers to specify properties of resources that they expect. These are defined
+in the `cuda::mr` namespace of libcu++, but RMM provides some convenience wrappers in
+`rmm/resource_ref.hpp`:
+ - `rmm::device_async_resource_ref` accepts a memory resource that provides synchronous allocation
```

Comment:
Do you mean `device_resource_ref` here? It looks like a duplicate of below.

Reply:
Yes typo. Made same error in libcudf.

Reply:
Fixed